spring-ai-core/src/main/java/org/springframework/ai/chat/client/advisor/observation/AdvisorObservationDocumentation.java

```diff
@@ -65,7 +65,7 @@ public String asString() {
 		ADVISOR_TYPE {
 			@Override
 			public String asString() {
-				return "spring.ai.chat.client.advisor.type";
+				return "spring.ai.advisor.type";
```

Comment:
@tzolov what do you think about this simplification of the naming for Advisor attributes?

Reply:
The advisors are only applicable in the context of ChatClient. 
But i guess it is enough to reflect this int the observation's name (e.g.  spring.ai.chat.client.advisor) and not in the attribute names?

Reply:
The related span is also nested under a "chat_client" parent span, so that might be enough. But if we should have it in the attribute name as well, I can revert this